configs/components/razor-server.rb

```diff
@@ -0,0 +1,93 @@
+component "razor-server" do |pkg, settings, platform|
```

Comment:
I would not block on this, but yes, I think we should be doing that for every vanagon project